crates/uv/src/commands/python/install.rs

```diff
@@ -144,20 +144,21 @@ pub(crate) async fn install(
 
     let reporter = PythonDownloadReporter::new(printer, downloads.len() as u64);
 
```

Comment:
Is it possible that the progress bar might be interleaved with the error message in line 178?

Reply:
Yes, it is. I manually altered one installation's SHA-256 checksum, and the progress bar was indeed interleaved by the hash mismatch error.
![image](https://github.com/user-attachments/assets/b69578a4-54c4-4d0a-9582-bd04f1e28fd9)

Reply:
Yeah I considered that at the time. Is it actually bad though? Otherwise you don’t get feedback until all downloads complete.

Reply:
In my experience, if you don't clear the progress bar before printing (using the process bar's printer), the terminal likes to eat part of the error message

Reply:
Ok, I'll move it after then.